Packs/RsaNetWitnessPacketsAndLogs/Integrations/RSANetWitnessPacketsAndLogs/RSANetWitnessPacketsAndLogs.yml

```diff
@@ -8,46 +8,31 @@ description: RSA NetWitness Logs and Packets decoders are responsible for the re
 configuration:
 - display: Server URL (e.g. http(s)://192.168.0.1)
   name: url
-  defaultvalue: ""
   type: 0
   required: true
-- display: Appliance Port - Logs(50102) / Packets(50104) / Concentrator (50105) / Broker (50103)
+- display: Appliance Port - Logs(50102) / Packets(50104) / Concentrator (50105) /
+    Broker (50103)
   name: port
-  defaultvalue: ""
   type: 0
   required: true
 - display: Username
   name: username
```

Comment:
please revert this section:
- display: Username
  name: username
  defaultvalue: admin
  type: 0
  required: false
  hidden: true
- display: Password
  name: password
  defaultvalue: ""
  type: 4
  required: false
  hidden: true
- display: Username
  name: user_creds
  type: 9
  required: false
  displaypassword: Password


  * In order to support backwards, we put the previous values in hidden and switched to type 9.

Reply:
done !